object-manipulation-3/main.js

```diff
@@ -0,0 +1,106 @@
+console.log('Lodash is loaded:', typeof _ !== 'undefined');
+// output: console.log the winner based on the point value of their hand\
+// set highscore to 0
+// create storage for winning player
+// create storage for player list
+// create deck of cards
+// shuffle deck
+// deal 2 cards per player
+// each time pull those cards from the deck
+// look at each player's score (sum of cards)
+// if player's points are higher than the current high score
+// make that player's pts the new high score
+// log the winner
+
+var highScore = 0;
+var winningPlayer = '';
+
+var playerArr = [
+  { name: 'Goku', hand: [], score: 0 },
+  { name: 'Master Roshi', hand: [], score: 0 },
+  { name: 'Krillin', hand: [], score: 0 },
+  { name: 'Bulma', hand: [], score: 0 }
+];
+
+var deck = [
+  { face: 'ace', rank: 'club', value: 11 },
+  { face: 'ace', rank: 'clover', value: 11 },
+  { face: 'ace', rank: 'diamond', value: 11 },
+  { face: 'ace', rank: 'heart', value: 11 },
+  { face: '2', rank: 'club', value: 2 },
+  { face: '2', rank: 'clover', value: 2 },
+  { face: '2', rank: 'diamond', value: 2 },
+  { face: '2', rank: 'heart', value: 2 },
+  { face: '3', rank: 'club', value: 3 },
+  { face: '3', rank: 'clover', value: 3 },
+  { face: '3', rank: 'diamond', value: 3 },
+  { face: '3', rank: 'heart', value: 3 },
+  { face: '4', rank: 'club', value: 4 },
+  { face: '4', rank: 'clover', value: 4 },
+  { face: '4', rank: 'diamond', value: 4 },
+  { face: '4', rank: 'heart', value: 4 },
+  { face: '5', rank: 'club', value: 5 },
+  { face: '5', rank: 'clover', value: 5 },
+  { face: '5', rank: 'diamond', value: 5 },
+  { face: '5', rank: 'heart', value: 5 },
```

Comment:
Try using a couple of loops to generate your deck from an array of ranks and an array of suits.